src/class-parsely.php

```diff
@@ -0,0 +1,1795 @@
+<?php
+/**
+ * This is the main class for Parsely
+ *
+ * @category   Class
+ * @package    Parsely
+ */
+class Parsely {
```

Comment:
The [file-level DocBlock summary](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#6-file-headers) is the **what** (no full stop). `@since` is the version when the file was created.

The [class-level DocBlock summary](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#2-classes) describes the **purpose** of the class (with a full stop). `@since` is the version the class was created, and in this case, an extra line to explain that it moved files.

`@category` is redundant and can be removed. Namespaces can serve a better purpose of programmatically organising code.

```suggestion
<?php
/**
 * Parsely class
 *
 * @package Parsely
 * @since 2.5.0
 */

/**
 * Holds most of the logic for the plugin.
 *
 * @internal This needs splitting up in the future.
 *
 * @since 1.0.0
 * @since 2.5.0 Moved from plugin root file to this file.
 */
class Parsely {
```